examples/99-advanced/planets.py

```diff
@@ -0,0 +1,115 @@
+"""
+.. _planets_example:
+
+3D Earth and Celestial Bodies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Creates high-resolution renderings of the Earth and the major celestial bodies
+in our solar system for astrodynamics applications.
+This example is inspired by `planet3D-MATLAB <https://github.com/tamaskis/planet3D-MATLAB>`_ .
+
+
+Note that this example author doesn't know anything about astronomy. There may
+be inaccuracies in the representation. The purpose of this example is for
+inspiration only.
+
+"""
+import pyvista
+from pyvista import examples
```

Comment:
We're starting to get a ton of examples in `examples`. Can we have something like:
```
from pyvista.examples import planets
```

And have all the planet textures within the `pyvista.examples.planets` module?

Reply:
Sounds nice. I will move globe data to the planets module too.

Reply:
See
https://github.com/pyvista/pyvista/commit/f4a54be7c71e0b254a242904c33a27102ab580ff